django/test/runner.py

```diff
@@ -581,6 +581,22 @@ def add_arguments(cls, parser):
             ),
         )
 
+    def log(self, msg, level=None):
+        """Log the given message at the given logging level."""
```

Comment:
Since this might not be obvious from the method implementation, it's probably worth spelling out with another sentence in the docstring: "A verbosity of 1 logs INFO (the default level) or above, and verbosity 2 logs all levels" (can go on a separate line).